lib/css-declaration-sorter.js

```diff
@@ -42,26 +44,35 @@ module.exports = {
adius, border-inline-start, border-inline-start-color, border-inline-start-style, border-inline-start-width, border-inline-end, border-inline-end-color, border-inline-end-style, border-inline-end-width, border-block-start, border-block-start-color, border-block-start-style, border-block-start-width, border-block-end, border-block-end-color, border-block-end-style, border-block-end-width, border-color, border-image, border-image-outset, border-image-repeat, border-image-slice, border-image-source, border-image-width, border-top-color, border-right-color, border-bottom-color, border-left-color, border-collapse, border-spacing, outline, outline-color, outline-offset, outline-style, outline-width, backdrop-filter, backface-visibility, background, background-attachment, background-blend-mode, background-clip, background-color, background-image, background-origin, background-position, background-position-x, background-position-y, background-repeat, background-size, box-shadow, isolation,  ruby-align, ruby-position, color, caret-color, font, font-family, src, font-feature-settings, font-kerning, font-language-override, font-size, font-optical-sizing, font-size-adjust, font-stretch, font-style, font-synthesis, font-variant, font-variant-alternates, font-variant-caps, font-variant-east-asian, font-variant-ligatures, font-variant-numeric, font-variant-position, font-weight, font-display, hyphens, initial-letter, initial-letter-align, letter-spacing, line-break, line-height, list-style, list-style-image, list-style-position, list-style-type, direction, text-align, text-align-last, text-decoration, text-decoration-color, text-decoration-line, text-decoration-skip, text-decoration-style, text-decoration-skip-ink, text-emphasis, text-emphasis-position, text-emphasis-color, text-emphasis-style, text-indent, text-justify, text-underline-position, text-orientation, text-overflow, text-rendering, text-shadow, text-size-adjust, text-transform, vertical-align, white-space, word-break, word-spacing, overflow-wrap, all, animation, animation-name, animation-duration, animation-fill-mode, animation-play-state, animation-timing-function, animation-delay, animation-iteration-count, animation-direction, mix-blend-mode, break-before, break-after, break-inside, page-break-before, page-break-after, page-break-inside, caption-side, clip-path, content, counter-increment, counter-reset, cursor, empty-cells, filter, image-orientation, image-rendering, mask, mask-clip, mask-image, mask-origin, mask-position, mask-repeat, mask-size, mask-type, opacity, perspective, perspective-origin, pointer-events, quotes, resize, scroll-behavior, scroll-snap-coordinate, scroll-snap-destination, scroll-snap-type, shape-image-threshold, shape-margin, shape-outside, tab-size, table-layout, text-combine-upright, touch-action, transition, transition-delay, transition-duration, transition-property, transition-timing-function, will-change, unicode-bidi, unicode-range, user-select, widows, writing-mod',
```

Comment:
I don't think we need a default value for this?

Reply:
yes this is a valid point